lib/static/components/controls/menu-bar.js

```diff
@@ -20,15 +22,20 @@ class MenuBar extends Component {
     render() {
         const {extraItems} = this.props;
 
-        if (isEmpty(extraItems)) {
+        const hasMenuBarPlugins = plugins.getLoadedConfigs()
+            .some(e => e.point === 'menu-bar');
```

Comment:
let's define available extension points inside separate file here - https://github.com/gemini-testing/html-reporter/tree/master/lib/constants. And then use them here and where specify `result` extension point.

Reply:
Done